syft/execution/plan.py

```diff
@@ -229,8 +229,9 @@ def __setattr__(self, name, value):
             for tensor_name, tensor in value.named_tensors():
                 self.__setattr__(f"{name}_{tensor_name}", tensor)
 
-    def __call__(self, *args):
+    def __call__(self, *args, **kwargs):
         """
+        Outdated!!
```

Comment:
Let's capture an issue to update the doc strings in `Plan` and associated classes. Then we can save that work for a later PR.